### DISCUSSION TITLE
How does the Wallabag plugin decide which articles to pull to your device? I can't see any real pattern to it or settings to change.

### DISCUSSION THREAD

**Discussion by davin900** [koreader/koreader#10853] [Q&A]
I know that I can choose filters based on tags. But I just set up Wallabag and have about 400 articles imported from Pocket and zero tags. I have the plugin limited to the default 30 articles.

So I successfully connected KOReader's Wallabag plugin to my instance of Wallabag and it pulled 30 articles... but not the most recent ones, not in alphabetical order, not just the oldest ones (I don't think).

How did it select those 30? And can I set it to just the 30 most recent unread somehow?

**Comment by Frenzie** [comment]
As I said in https://github.com/koreader/koreader/discussions/10847#discussioncomment-6836308 afaict that's simply what it does. Wallabag returns the newest, followed by older, and so forth.

What you left out there is "from Pocket", so that's the most logical distinction. For whatever reason the order from the Pocket import wasn't in the correct order of time added but some other random order. You should be able to verify that in the web interface.
